jax/_src/api.py

```diff
@@ -153,8 +153,13 @@ def jit(
   abstracted_axes: Any | None = None,
   compiler_options: dict[str, Any] | None = None,
 ) -> pjit.JitWrapped:
-  """Sets up ``fun`` for just-in-time compilation with XLA.
+  """A JAX transformation that sets up ``fun`` for just-in-time compilation with XLA.
+  
+  Learn more in the Just-in-time compilation tutorial [1] and the Quickstart [2].
 
+  [1] https://jax.readthedocs.io/en/latest/jit-compilation.html
+  [2] https://jax.readthedocs.io/en/latest/quickstart.html#just-in-time-compilation-with-jax-jit
```

Comment:
I'm waiting on the readthedocs build, but I don't think this will render well in HTML. Perhaps we should use the sphinx hyperlink syntax? See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
(I was going to wait to see the results of the build, but I saw you opening several other PRs using the same pattern – you should double-check that the rendered output looks OK before doing any more like this)

Reply:
SGTM!

Reply:
Forgot to set them as DRAFT PRs.